src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -4763,7 +4763,7 @@ public static Task EnsureDotnetInstallExistsAsync()
     <add key=""benchmarks-extensions"" value=""https://dotnetfeed.blob.core.windows.net/aspnet-extensions/index.json"" />
     <add key=""benchmarks-aspnetcore-tooling"" value=""https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore-tooling/index.json"" />
     <add key=""benchmarks-entityframeworkcore"" value=""https://dotnetfeed.blob.core.windows.net/aspnet-entityframeworkcore/index.json"" />
-    <add key=""benchmarks-nuget"" value=""https://api.nuget.org/v3/index.json"" />
+    <add key=""dotnet-public"" value=""https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json"" />
```

Comment:
@mmitche How are the packages added to this feed? We benchmark apps that require special packages/versions, and they are not part of the CI process. Or did they get discovered from any <PackageReference> ?